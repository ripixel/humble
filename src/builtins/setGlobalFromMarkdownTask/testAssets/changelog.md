# Changelog

## [1.0.0] - 2025-01-01
- Initial release

## [1.1.0] - 2025-02-01
- Added new features
