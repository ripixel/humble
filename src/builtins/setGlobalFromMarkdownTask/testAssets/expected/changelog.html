<h1>Changelog</h1>
<h2>[1.0.0] - 2025-01-01</h2>
<ul>
  <li>Initial release</li>
</ul>
<h2>[1.1.0] - 2025-02-01</h2>
<ul>
  <li>Added new features</li>
</ul>
